packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -682,14 +717,21 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
     schemaComposer.add(SortDirectionEnum);
     schemaComposer.add(StringScalarInputType);
     schemaComposer.add(BooleanScalarInputType);
+    schemaComposer.add(JSONScalarInputType);
     schemaComposer.add(createInputTypeForScalar(GraphQLInt));
     schemaComposer.add(createInputTypeForScalar(GraphQLFloat));
 
     schemaComposer.forEach((tc: NamedTypeComposer<any>) => {
       const namedType = tc.getType();
-      if (isScalarType(namedType) && !isSpecifiedScalarType(namedType)) {
+      if (isScalarType(namedType) && !isSpecifiedScalarType(namedType) && !isSpecifiedGraphbackScalarType(namedType)) {
         schemaComposer.add(createInputTypeForScalar(namedType));
       }
```

Comment:
```suggestion
      return;
      }
```

let's short-circuit the flow to avoid the checks below.